pkg/api/aim2/dao/repositories/run.go

```diff
@@ -407,6 +407,10 @@ func (r RunRepository) SearchRuns(
 		).
 		Order("row_num DESC")
 
+	// if len(req.ExperimentNames) != 0 {
```

Comment:
seems like this should be uncommented

Reply:
once uncommented, there are many test failures. trying to figure out the right encantation...